src/clj_holmes/output/sarif.clj

```diff
@@ -1,20 +1,21 @@
 (ns clj-holmes.output.sarif)
 
-(defn ^:private result->sarif-rule [{:keys [message severity] :as rule}]
-  (let [rule (select-keys rule [:id :name :properties])
-        sarif-rule {:fullDescription      {:text message}
-                    :shortDescription     {:text message}
+(defn ^:private result->sarif-rule [{:keys [id name message severity] :as rule}]
+  (let [rule (select-keys rule [:id :properties])
+        sarif-rule {:name                 id
+                    :shortDescription     {:text name}
+                    :fullDescription      {:text message}
                     :help                 {:text message}
                     :defaultConfiguration {:level severity}}]
     (merge rule sarif-rule)))
 
 (defn ^:private sarif-boilerplate [results]
-  {:$schema "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Documents/CommitteeSpecifications/2.1.0/sarif-schema-2.1.0.json"
+  {:$schema "https://www.schemastore.org/schemas/json/sarif-2.1.0-rtm.5.json"
    :version "2.1.0"
    :runs    [{:tool
               {:driver {:name           "clj-holmes"
-                        :informationUri "https://github.com/mthbernardes/clj-holmes"
-                        :version        (System/getProperty "clj-holmes.version")
+                        :informationUri "https://github.com/clj-holmes/clj-holmes"
+                        :version        (nth (read-string (slurp "project.clj")) 2)
```

Comment:
Since the property `clj-holmes.version` is available only in build time you could just add it to a `def` in this namespace instead of reading the `project.clj` (there are plans to migrate from lein to deps)

Reply:
thanks for the suggestion. added a `def` variable.